src/markdown/privacy/privacy.md.d.ts

```diff
@@ -0,0 +1,5 @@
+export { default } from '*.md'
+export const metadata = {
+  version: string,
+  last_update_date: string,
+}
```

Comment:
You are specifying a frontmatter with this, but your md file doesn't have one. Look at terms.md

```
---
version: 1.2
last_update_date: September, 2024
---
```

Reply:
Afaik we are not using those anywhere in the app like we do for terms so I will remove the `.d.ts` file